kettle-plugins/hbase/src/main/java/org/pentaho/big/data/kettle/plugins/hbase/mapping/MappingEditor.java

```diff
@@ -766,6 +766,9 @@ public Mapping getMapping( boolean performChecksAndShowGUIErrorDialog, List<Stri
     HBaseService hBaseService = null;
     try {
       hBaseService = m_configProducer.getHBaseService();
+      if (hBaseService == null ) { // Backlog-32244; If we don't have a service don't bother trying to map it.
```

Comment:
Solution is fine; put a space in after the ( here to make Sonar happy and we can merge it.